types/recurly__recurly-js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for non-npm package @recurly/recurly-js 4.23
+// Type definitions for non-npm package @recurly/recurly-js 4.25
 // Project: https://github.com/recurly/recurly-js
 // Definitions by: Christopher Rogers <https://github.com/chrissrogers>
```

Comment:
If the owners of the actual Recurly project plan to update these types from @recurly-integrations, please add that account as an owner here. These kinds of PRs aren’t meaningfully reviewable, so we may as well get the signal that they can be fast-tracked due to the owner of the project taking responsibility for updates to the types.